gtsam/base/VectorSpace.h

```diff
@@ -411,6 +420,8 @@ struct DynamicTraits {
 
   /// @name Lie Group
   /// @{
+  using LieAlgebra = Dynamic;
+    
   static TangentVector Logmap(const Dynamic& m, ChartJacobian H = {}) {
```

Comment:
Maybe replace `Dynamic` with `LieAlgebra`?

Reply:
No, that does not make sense I think.